pkg/operator/ceph/file/filesystem.go

```diff
@@ -156,6 +176,9 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(". %v", err),
 			)
 		}
+		if err := SetPoolSize(context, clusterName, f.Name, f.dataPools, f.metadataPool); err != nil {
+			return errors.Wrapf(err, "failed to set pools size")
```

Comment:
```suggestion
			return errors.Wrap(err, "failed to set pools size")
```

Reply:
Thanks, since there is no format specifier, `Wrap` will be used.